app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryComfortableGridHolder.kt

```diff
@@ -48,6 +48,11 @@ class LibraryComfortableGridHolder(
             isVisible = item.downloadCount > 0
             text = item.downloadCount.toString()
         }
+        // Update the source language and its visibility
+        with(binding.languageText) {
+            isVisible = item.sourceLanguage != ""
+            text = "${item.sourceLanguage}"
```

Comment:
```suggestion
            text = item.sourceLanguage
```

Same for the other instances that are using string interpolation.

Reply:
Adjusted the string interpolation, the badges are back to the round rectangle style (didn't see that earlier haha) and changed the badge color to tertiary to improve readability in list view. It was getting pretty confusing with the same color.